<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addmachine</class>
 <widget class="QDialog" name="addmachine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a Machine</string>
  </property>
  <widget class="QLabel" name="port_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>SSH Port</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_line">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>80</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="host_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Host/IP</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="host_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="password_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="key_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>221</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>32765</number>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="user_label">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>40</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="user_line">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="key_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Key</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browse_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>420</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="port_line">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="configure_button">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Configure</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>420</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QLabel" name="tunnel_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>SSH Tunneling</string>
   </property>
  </widget>
  <widget class="QLabel" name="name_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="portforward_check">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>280</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Port Forward?</string>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="localport_line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>350</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="localhost_line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="localhost_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>320</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Local Host</string>
   </property>
  </widget>
  <widget class="QLabel" name="localport_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>350</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Local Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="remoteport_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>410</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Remote Port</string>
   </property>
  </widget>
  <widget class="QLabel" name="remotehost_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>380</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Remote Host</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="remotehost_line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="remoteport_line">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>410</y>
     <width>71</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="local_forwarding">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Local Forwarding</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="remote_forwarding">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>350</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Remote Forwarding</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>name_line</tabstop>
  <tabstop>host_line</tabstop>
  <tabstop>port_line</tabstop>
  <tabstop>user_line</tabstop>
  <tabstop>password_line</tabstop>
  <tabstop>key_line</tabstop>
  <tabstop>browse_button</tabstop>
  <tabstop>configure_button</tabstop>
  <tabstop>portforward_check</tabstop>
  <tabstop>add_button</tabstop>
  <tabstop>clear_button</tabstop>
  <tabstop>localhost_line</tabstop>
  <tabstop>localport_line</tabstop>
  <tabstop>remotehost_line</tabstop>
  <tabstop>remoteport_line</tabstop>
  <tabstop>local_forwarding</tabstop>
  <tabstop>remote_forwarding</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
